models/classes/class.QtiTestConverter.php

```diff
@@ -301,7 +301,7 @@ private function arrayToComponent(array $testArray, QtiComponent $parent = null,
 
             if (! empty($compName)) {
                 $reflector = new ReflectionClass($compName);
-                $component = $this->createInstance($reflector, $testArray);
+                $component = $this->createInstance($reflector, $testArray, $parent);
```

Comment:
- [ ] @shpran the $parent is pointed only for recursiveness. By passing it with the intent of just building properly the branching rules, we are also making this call on line 335 https://github.com/oat-sa/extension-tao-testqti/pull/2695/files#diff-18edd2312f87399ee56640e3c7446c6d386b4b47f23e4e46eb797b7c732a4e6eR335 to be executed. I think this may bring side effects. Can you please double check this is not causing any performance or other issues?